kubernetes/zenko/charts/cloudserver/templates/_helpers.tpl

```diff
@@ -48,3 +48,11 @@ Increases the number of cloudserver replicas by the replicaFactor value
 {{- printf "%d" $factor }}
 {{- end -}}
 
+{{/*
+Create the default redis sentinels hosts string
```

Comment:
Can there be a global helper that we can use across the charts to avoid code duplication?

Reply:
This can potentially be at the `zenko` level helpers but we've always tried to keep the charts as individually usable as possible. That being said, it's possible to eventually have a convergence of cloudserver and backbeat charts into zenko level.

Reply:
Ok. I think we will see more coming this way.

Reply:
Yeah I think that convergence is probably inevitable with the amount of similar configs we have across these two charts. It'll also likely make it easier with getting Zenko into kubernetes upstream repos like helm/charts.